vectordatasource/transform.py

```diff
@@ -4224,6 +4254,46 @@ def _normalize_ca_netref(network, ref):
     return network, ref
 
 
+def _normalize_cn_netref(network, ref):
+    if ref.startswith('S'):
+        network = 'CN:expressways:regional'
+
+    elif ref.startswith('G'):
+        network = 'CN:expressways'
+
+    elif ref.startswith('X'):
+        network = 'CN:jx-roads'
+
+    elif network == 'CN-expressways':
+        network = 'CN:expressways'
+
+    elif network == 'CN-expressways-regional':
+        network = 'CN:expressways:regional'
+
+    elif network == 'JX-roads':
+        network = 'CN:jx-roads'
```

Comment:
These look to all be "region" roads in the [Jiangxi](https://en.wikipedia.org/wiki/Jiangxi) province and https://overpass-turbo.eu/s/zbf. Should the convention be `CN:JX`?

Reply:
Yup, sounds good. Updated in 89a1881.